doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -134,6 +134,15 @@ Bluetooth samples
 
     * Increased the main stack size from 1024 to 1536 bytes.
 
+  * :ref:`peripheral_uart` sample:
+
+    * Fixed double attempt to free the UART buffer.
```

Comment:
I think we could elaborate this a bit. Is the double attempt something that was in the sample and it is not there anymore? Not sure how to interpret this.

Reply:
Ok. I wrote the longer paragraph about my changes :)